### PR TITLE
Add jq as requirement

### PR DESCRIPTION
**Pull request by dullookaran**
## Summary of the changes / Why this is an improvement

Added the link with jq installation as this is required as well

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Review by SStorm — APPROVED** [review]
Thanks, normally this would be approved by @matthijskrul but since he's on holiday I think he won't mind :) 

Also, the description text (in the PR description) should be in the right section.

File: docs/edge.rst [hunk 1]
```diff
@@ -165,7 +165,7 @@ and you will have to install them to proceed. (We recommend `Helm`_ for
 tracking and installing dependencies on Kubernetes.)
 
 .. NOTE::
-    You must have `wget`_ installed for the script to function.
+    You must have `wget` & 'jq'_ installed for the script to function.
```
Comment (hammerhead) [inline docs/edge.rst]: Aren't the links broken that way?

```suggestion
    You must have `wget`_ & `jq`_ installed for the script to function.
```

Comment (jayeff) [reply docs/edge.rst]: good catch 🙌

**Review by kojinkai — APPROVED** [review]

File: docs/edge.rst (hunk 1 above)
Comment (jayeff) [reply docs/edge.rst]: fixed

**Pull request merged**